.travis.yml

```diff
@@ -4,7 +4,8 @@ os: linux
 dist: bionic
 jdk:
   - openjdk8
-
+  - openjdk11
```

Comment:
I think that doing this will cause Travis to build and test the JAR with Java 11 - but we don't actually distribute the JAR file built with Java 11 - only Java 8. So I'm not sure that testing with openjdk11 really buys us anything - but it will slow down all the builds.

Reply:
Yeah, that's true, it will. I guess, at some point, we may want to test the jar we're distributing in more environments. But, I'll take this out for now.